autogluon/utils/tabular/ml/models/xgboost/hyperparameters/searchspaces.py

```diff
@@ -0,0 +1,68 @@
+""" Default hyperparameter search spaces used in XGBoost Boosting model """
+from .......core import Real, Int
+from ....constants import BINARY, MULTICLASS, REGRESSION
+
+DEFAULT_NUM_BOOST_ROUND = 10000  # default for HPO
+
+
+def get_default_searchspace(problem_type, num_classes=None):
+    if problem_type == BINARY:
+        return get_searchspace_binary_baseline()
+    elif problem_type == MULTICLASS:
+        return get_searchspace_multiclass_baseline(num_classes=num_classes)
+    elif problem_type == REGRESSION:
+        return get_searchspace_regression_baseline()
+    else:
+        return get_searchspace_binary_baseline()
+
+
+def get_searchspace_multiclass_baseline(num_classes):
```

Comment:
generally we want defaults to be equal to the default parameters, so for example if `'learning_rate': 0.03` with hpo off, then `'learning_rate': Real(lower=5e-3, upper=0.2, default=0.03, log=True)` should be set so default learning rates are equal.

Reply:
Ok. I set them as equal.